markdown/org/docs/patterns/jane/cutting/en.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Jane Shift: Cutting Instructions"
+---
+
+The best way to get a truly straight line for all the straight seams in Jane is to pull a thread. This is done by snipping into the fabric at the right measurement and then carefully pulling on the single thread that the cut exposes. Once pulled all the way out of the fabric it creates a void that is easily followed to cut a straight line. 
+
+There are two different ways to cut Jane, with or without a shoulder seam. The pattern doesn’t include a back and front to save on paper.
+
+If you are cutting Jane with a shoulder seam you:
+- cut one body part with front neckline
+- cut one body part with back neckline
+- cut gusset line in front and back part
```

Comment:
You might want to be consistent with the terminology used for this part. In some sections of the documentation these are called "gussets" and in other parts they are called "gores". Having one standard term might help to avoid confusion.

A diagram or drawing of the how the garment is constructed also might help to make things clearer. 

A diagram or drawing of the body part, specifically pointing out the gores/gussets might also be helpful, to help the customer realize that the triangles are the gores/gussets.

Reply:
Ive rewritten the file to say gore and gusset in the right places.